### PR TITLE
Rotate holography beams when linmos'ing

### PR DESCRIPTION
**Pull request by tjgalvin**
The ASKAP third-axis (physical rotation) and rotation of the footprint (electronically rotated) together will place the instrumental response onto the sky in a particular direction. If the science field has a different alignment than the data collected during holography, there needs to be an alignment process when coadding images in order to properly apply the constrained holography beams to the same direction as the science field. 

This pull request aims to:
- pull out the pol axis from a measurement set
- carry this information through the pipeline
- pass it over to linmos, deriving the appropriate rotation using *an assumed* parameterisation of the holography beam alignment

This aims to address #86

### CODE REVIEW

File: flint/summary.py [hunk 1]
```diff
@@ -89,6 +92,26 @@ def with_options(self, **kwargs) -> FieldSummary:
         return FieldSummary(**prop)
 
 
+def _get_pol_axis_as_rad(ms: Union[MS, Path]) -> float:
```
Comment (tjgalvin) [inline flint/summary.py:95]: This is the function that attempts to get the most correct orientation. I am not sure whether I want to force an error to be raised if the instrument receptor angle is missing, or whether this is ok. This is a non-standard keyword.

File: flint/summary.py [hunk 2]
```diff
@@ -126,11 +149,16 @@ def add_ms_summaries(
     hour_angles = centre_altaz.az.to(u.hourangle)
     elevations = centre_altaz.alt.to(u.deg)
 
+    # The INSTRUMENT_RECEPTOR_ANGLE comes from fixms and is
```
Comment (tjgalvin) [inline flint/summary.py:152]: Once it has been added to the field summary, then it is passed through to the linmos tasks.

File: flint/prefect/flows/continuum_pipeline.py [hunk 3]
```diff
@@ -251,98 +253,104 @@ def process_science_fields(
     fits_beam_masks = None
 
     for current_round in range(1, field_options.rounds + 1):
```
Comment (tjgalvin) [inline flint/prefect/flows/continuum_pipeline.py:255]: Much of this has changed simply from the `with tags` context manager. But do note passing in the field summary to the appropriate linmos tasks

File: flint/coadd/linmos.py [hunk 4]
```diff
@@ -259,6 +267,19 @@ def generate_linmos_parameter_set(
     # attribute drops the parent directory (absolute directory).
     parent_dir = linmos_names.image_fits.parent
 
+    # Work out the appropriate rotation to provide linmos. This should
```
Comment (tjgalvin) [inline flint/coadd/linmos.py:270]: This is the part that most worries me. I think it is correct. But, who knows. Maybe you. You pirate.

File: flint/summary.py (hunk 1 above)
Comment (AlecThomson) [reply flint/summary.py:95]: Depends on how much we want to be locked into FixMS. Will the observatory MSs ever have correct feeds?

**Review by AlecThomson — APPROVED** [review]
Looking all good to me. We may want to consider dumping the field summary to FITS headers at some point...

File: flint/summary.py (hunk 1 above)
Comment (tjgalvin) [reply flint/summary.py:95]: At this point I would not expect them to.

If they did I would they would also rotate the correlations to the sky as well as update the receptor angle. if so, I think the problem kind of drops out even if we keep fixms in out respective codez.

File: flint/coadd/linmos.py [hunk 5]
```diff
@@ -259,6 +267,19 @@ def generate_linmos_parameter_set(
...
+    # be the differential pol. axis. rotation between the science field
+    # and the holography. At the moment this holography rotation is
+    # unknown to us (not stored in the cube header).
+    alpha = None
+    if pol_axis:
+        logger.info(
+            f"The constant assumed holography pol_axis + footprint rotation is: {EXPECTED_HOLOGRAPHY_ROTATION_CONSTANT_RADIANS:.4f} radians"
```
[the remark this answers is not in the document]
Comment (tjgalvin) [reply flint/coadd/linmos.py:277]: Perhaps it is a naming thing. I have no tie to the log message, other than just putting something there for reporting. Thoughts on the alternative?

**Pull request merged**